src/machines/modelingMachine.ts

```diff
@@ -207,9 +214,9 @@ export const modelingMachine = createMachine(
               target: 'Sketch',
               cond: 'Selection is one face',
               actions: [
+                'set sketch metadata',
                 'sketch mode enabled',
                 'edit mode enter',
-                'set sketchPathToNode',
               ],
```

Comment:
Ohh okay coming together now, so the edit sketch was kind working but the animation was broken it was just a mater of changing the action order for the "Enter sketch" event
https://github.com/KittyCAD/modeling-app/pull/796/commits/5fb915d3dc67cfb36f0c068594e21b29756c5aa4